ydb/public/lib/ydb_cli/common/progress_bar.cpp

```diff
@@ -62,12 +62,12 @@ void TProgressBar::Render()
     output += TString("█") * filledBarLen;
     output += TString("░") * (barLen - filledBarLen);
     output += outputEnd;
-    Cout << output;
+    Cerr << output;
```

Comment:
`import file` command outputs debug info in stdout and uses progress bar. I think mixing isn't good, better fix output in commands using progress bar too